fpga_interchange/prjxray_db_reader.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2021 The SymbiFlow Authors.
+#
+# Use of this source code is governed by a ISC-style
+# license that can be found in the LICENSE file or at
+# https://opensource.org/licenses/ISC
+#
+# SPDX-License-Identifier: ISC
+
+import os
+import json
+
+
+class prjxray_db_reader:
+    def __init__(self, timing_dir):
+        self.timing_dir = timing_dir
+
+    def extract_data(self):
+        return_dict = {}
+        for i, _file in enumerate(os.listdir(self.timing_dir)):
+            if not os.path.isfile(os.path.join(
+                    self.timing_dir, _file)) or 'tile_type_' not in _file:
+                continue
+            with open(os.path.join(self.timing_dir, _file), 'r') as f:
+                tile_data = json.load(f)
+            print(_file, tile_data["tile_type"])
+            tile_name = tile_data['tile_type']
+            tile_dict = {}
+            tile_dict['wires'] = {}
+            for name, data in tile_data['wires'].items():
+                if data is None:
+                    continue
+                wire_name = name
+                tile_dict['wires'][wire_name] = (
+                    tuple([float(data['res']) * 1e-6] * 6),
```

Comment:
it'd probably be cleaner to have the scale factor applied to R, C and delay as constants at the top of the file, as well as a note about the unit you are scaling to/from (the prjxray docs are a bit unclear on this)